Detectors/TPC/simulation/src/Digitizer.cxx

```diff
@@ -106,7 +106,7 @@ void Digitizer::process(const std::vector<o2::tpc::HitGroup>& hits,
           LOG(warning) << "Skipping electron with driftTime " << driftTime << " from hit at time " << hitTime;
           continue;
         }
-        const float absoluteTime = eleTime + (mEventTime - mOutputDigitTimeOffset); /// in us
+        const float absoluteTime = eleTime + mTDriftOffset + (mEventTime - mOutputDigitTimeOffset); /// in us
```

Comment:
Hi @wiechula could you please confirm that this is enough to impose the drift time offset (defined in \mus) in the simulation?

Reply:
Hi @shahor02 , yes this should be the only place.